tools/test_common.rb

```diff
@@ -0,0 +1,14 @@
+if ENV["BUILDKITE"]
+  module Minitest
+    def self.plugin_rails_ci_junit_format_test_report_for_buildkite_init(*)
+      require "minitest/reporters"
```

Comment:
What do you think of moving to require before defining a module? 
It seems like there is no need to require inside the method now.